src/components/textField/ValidationMessage.tsx

```diff
@@ -24,11 +28,24 @@ const ValidationMessage = ({
   const relevantValidationMessage = getRelevantValidationMessage(validationMessage, context.failingValidatorIndex);
   const showValidationMessage = !context.isValid || (!validate && !!validationMessage);
 
-  return (
-    <Text testID={testID} $textDangerLight style={style}>
-      {showValidationMessage ? relevantValidationMessage : ''}
-    </Text>
-  );
+  const renderMessage = () => {
+    return (
+      <Text testID={testID} $textDangerLight style={style}>
+        {showValidationMessage ? relevantValidationMessage : ''}
+      </Text>
+    );
+  };
+
+  if (validationIcon) {
```

Comment:
Check the example screen with long validation messages.. 
There's on example when the layout breaks (It might be the example fault)

Reply:
Long validation message breaks the layout even without the icon or helperText. I'll fix it